src/lib/importer/hoyoLabFormatParser.tsx

```diff
@@ -0,0 +1,334 @@
+        stat: SubStats
+        value: number
+        addedRolls: number
+      }[]
+      verified: boolean
+    }[]
+  } = {
+    metadata: {
+      trailblazer: 'Stelle',
+      current_trailblazer_path: 'Destruction',
+    },
+    characters: [],
+    relics: [],
+  }
+  for (let i = 0; i < json.data.avatar_list.length; i++) {
+    const character = json.data.avatar_list[i]
+    if (character.equip != null) {
+      output.characters.push({
+        characterEidolon: character.rank,
+        characterId: (character.id).toString(),
+        characterLevel: character.level,
+        lightCone: (character.equip.id).toString(),
+        lightConeLevel: character.equip.level,
+        lightConeSuperimposition: character.equip.rank,
+      })
+    } else {
+      output.characters.push({
+        characterEidolon: character.rank,
+        characterId: (character.id).toString(),
+        characterLevel: character.level,
+        lightCone: null,
+        lightConeLevel: 80,
+        lightConeSuperimposition: 1,
+      })
+    }
+    for (let j = 0; j < character.relics.length; j++) {
+      const relic = character.relics[j]
+      const substats: {
+        stat: SubStats
+        value: number
+        addedRolls: number
+      }[] = []
+      for (let k = 0; k < relic.properties.length; k++) {
+        const substat = {
+          stat: getStat(relic.properties[k].property_type),
+          value: readValue(relic.properties[k].value),
+          addedRolls: relic.properties[k].times - 1,
+        }
+        substats.push(substat)
+      }
+      output.relics.push({
+        enhance: relic.level,
+        equippedBy: (character.id).toString(),
+        grade: relic.rarity,
+        id: uuidv4(),
+        part: getSlot(relic.pos),
+        set: getSet(relic.id),
+        main: {
+          stat: getStat(relic.main_property.property_type),
+          value: readValue(relic.main_property.value),
+        },
+        substats: substats,
+        verified: false,
+      })
+    }
+    for (let j = 0; j < character.ornaments.length; j++) {
+      const relic = character.ornaments[j]
+      const substats: {
+        stat: SubStats
+        value: number
+        addedRolls: number
+      }[] = []
+      for (let k = 0; k < relic.properties.length; k++) {
+        const substat = {
+          stat: getStat(relic.properties[k].property_type),
+          value: readValue(relic.properties[k].value),
+          addedRolls: relic.properties[k].times - 1,
+        }
+        substats.push(substat)
+      }
+      output.relics.push({
+        enhance: relic.level,
+        equippedBy: (character.id).toString(),
+        grade: relic.rarity,
+        id: uuidv4(),
+        part: getSlot(relic.pos),
+        set: getSet(relic.id),
+        main: {
+          stat: getStat(relic.main_property.property_type),
+          value: readValue(relic.main_property.value),
+        },
+        substats: substats,
+        verified: false,
+      })
+    }
```

Comment:
This kinda looks identical to `character.relics`, does it work if we do something like 

```js
for (const relic of [...character.relics, ...character.ornaments]) {
  ...
}
```